core/src/main/scala/scala/pickling/pickler/Nothing.scala

```diff
@@ -0,0 +1,16 @@
+package scala.pickling.pickler
+
+import scala.pickling._
+import PicklingErrors.NothingIsNotUnpicklable
+
+trait NothingPicklers {
+  implicit object NothingPicklerUnpickler extends AbstractPicklerUnpickler[Nothing] {
+    override def tag: FastTypeTag[Nothing] = FastTypeTag.Nothing
+    /** Impossible to call in Scala, no value can be typed [[Nothing]] */
+    override def pickle(picklee: Nothing, builder: PBuilder): Unit = ???
```

Comment:
If we do need to have it, then we should write a comment about how it's impossible to call it...